### DISCUSSION TITLE
Log collection Chinese garbled code

### DISCUSSION THREAD

**Discussion by BertOnline** [apache/skywalking#10067] [Q&A] [answered]
![Snipaste_2022-12-01_10-37-55](https://user-images.githubusercontent.com/17882970/204966765-7b790fe7-4c52-4d91-ace6-d49844a2818a.png)

Dear community, what is the cause of Chinese garbled code in log collection? Is my configuration wrong?

**Comment by BertOnline** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/17882970/204969631-ed9ccb7c-12e7-4242-8d10-7618944383a7.png)

I found that the data uploaded to elasticsearch by the oap server is Chinese garbled code

**Reply by wu-sheng** [reply]
We don't encode the content, please check whether your logs are UTF-8 encoded.

**Reply by BertOnline** [reply]
> We don't encode the content, please check whether your logs are UTF-8 encoded.

OK，Thanks

**Reply by BertOnline** [reply]
Solution to garbled code problem: jvm adds - Dfile. encoding=UTF-8 to my own application.
